src/cider/nrepl/middleware/util/instrument.clj

```diff
@@ -246,6 +248,36 @@
   ;; Don't use `postwalk` because it destroys previous metadata.
   (walk-indexed #(tag-form %2 breakfunction) form))
 
+(defn transfer-break-conditions
```

Comment:
This part is a little tricky. Can we leave it out for now? 

Firstly, it shouldn't be here. Doing something like this is the job of `debug.clj`, not `instrument.clj`.

Second, I realise you need this so that your Emacs command will work, but I still want to discuss what's the best interface for this feature.